src/test/resources/recordings/com.microsoft.azure.spark.tools.processes.SparkBatchJobRemoteProcessScenario/mappings/livy_batches_261-0a40e506-d70c-4faf-89c4-c52eeff08582.json

```diff
@@ -1,8 +1,8 @@
 {
```

Comment:
Do we need to keep these recording json files?

Reply:
They are used for replay in UT.